resources/dicts/patrols/general/medcat.json

```diff
@@ -479,147 +838,345 @@
+                    "daring",
+                    "shameless",
+                    "oblivious"
+                ],
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["app1"],
+                        "mutual": false,
+                        "values": ["comfort", "respect", "trust"],
+                        "amount": -5
+                    }
+                ]
+            },
+            {
+                "text": "s_c stares at the apprentice with undisguised envy, desperately wishing {PRONOUN/s_c/subject} {VERB/s_c/were/was} the one to receive the vision. Glancing at the older cat nervously, app1 mumbles that it wasn't important.",
+                "exp": 0,
+                "weight": 20,
+                "stat_trait": [
+                    "ambitious", 
+                    "insecure", 
+                    "vengeful"
+                ],
+                "relationships": [
+                    {
+                        "cats_to": ["r_c"],
+                        "cats_from": ["app1"],
+                        "mutual": true,
+                        "values": ["respect", "trust", "comfort", "platonic"],
+                        "amount": -5
+                    }
+                ]
+            },
+            {
+                "text": "s_c stares at the apprentice with a cold look, telling {PRONOUN/app1/object} to figure out {PRONOUN/app1/poss} dreams {PRONOUN/app1/self}, and that its doubtful someone like app1 would have a vision.",
+                "exp": 0,
+                "weight": 10,
+                "can_have_stat":["p_l"],
+                "stat_trait": [
+                    "ambitious", 
+                    "vengeful",
+                    "troublesome",
+                    "bloodthirsty",
+                    "cold",
+                    "strict",
+                    "competitive",
+                    "rebellious",
+                    "arrogant"
+                ],
+                "relationships": [
+                    {
+                        "cats_to": ["s_c"],
+                        "cats_from": ["app1"],
+                        "mutual": false,
+                        "values": ["respect", "trust", "comfort", "platonic"],
+                        "amount": -10
+                    },
+                    {
+                        "cats_to": ["s_c"],
+                        "cats_from": ["app1"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "At first app1 doesn't speak up, however p_l notices the apprentice being distracted, and gently asked what was wrong. Instead of opening up though, app1 shuts down, refusing to talk about what was bothering {PRONOUN/app1/object} and just says that they should focus on gathering herbs.",
```

Comment:
```suggestion
                "text": "At first app1 doesn't speak up, however p_l notices the apprentice is distracted, and gently asks what's wrong. Instead of opening up though, app1 shuts down, refusing to talk about what was bothering {PRONOUN/app1/object} and just says that they should focus on gathering herbs.",
```